api/v1beta1/vaultconnection_types.go

```diff
@@ -22,7 +22,7 @@ type VaultConnectionSpec struct {
 	CACertSecretRef string `json:"caCertSecretRef,omitempty"`
 	// SkipTLSVerify for TLS connections.
 	// +kubebuilder:default=false
-	SkipTLSVerify bool `json:"skipTLSVerify"`
+	SkipTLSVerify bool `json:"skipTLSVerify,omitempty"`
```

Comment:
I think this one we want to keep as-is. We made this change under #527 - to avoid an issue with Rancher.

Reply:
We can certainly do that, especially since this has been in place since v0.4.3.

FWIW in my testing this behaves the same with or without `omitempty` when there's a default set. If the parameter is set to true/false in the CRD spec it comes back on a get. And if the parameter is unspecified it comes back as `false` on a get. Which I think satisfies #527 and #278.